misc/extra/pom.xml

```diff
@@ -72,6 +72,16 @@
             <artifactId>cdk-smiles</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cdk-smarts</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cdk-silent</artifactId>
```

Comment:
Is there a way to remove this dependency - note for the QueryChemObjects you can just pass null for the builder.

Reply:
Hi @johnmay , thanks for reviewing!
I need the SilentChemObjectBuilder in the tests to parse SMILES and be able to add implicit hydrogens. Should I maybe add the test scope in the pom? Or I could switch to DefaultChemObjectBuilder which would apparently not require this extra dependency.

Reply:
Both OK but I would try and add ``<scope>test</scope>``

Reply:
Added test scope in the new commit. Would it be ready to merge now?